client/src/screens/SignUp/SignUp.scss

```diff
@@ -1,43 +1,40 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap');
-
-
 .auth-form {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    padding-top: 80px;
-    font-family: 'Roboto', sans-serif;
-    h4 {
-        align-self: center;
-        text-align: center;
-        width: max-content;
-        font-weight: 500;
-    }
-    label {
-        font-weight: 300;
-        font-size: 14px;
-    }
-    input {
-        width: 100%;
-        height: 28px;
-        border: solid black 1px;
-        padding-left: 7px;
-    }
-    .input-wrapper {
-        width: 100%;
-        margin-top: 7px;
-    }
-    h6 {
-        align-self: center;
-    }
-    a {
-        color: black;
-        font-size: 12px;
-        font-style: italic;
-        align-self: center;
-    }
+  display: flex;
```

Comment:
spoke too soon 💔

Reply:
Haha Idk how that got in there, but I will alphabetize the sign up scss.